SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -67,17 +56,15 @@ The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS Unreal
     If enabled, shuts down running deployments when you close the Unreal Editor.
 
     * **Snapshot path**</br>
-    Use this to specify the filepath of your Unreal GDK snapshot. If you leave this empty, it defaults to `<ProjectFileLocation>/../spatial/snapshots`.
+    Use this to specify the filepath of your snapshot. If you leave this empty, it defaults to `<GameRoot>/../spatial/snapshots`.
 
     * **Snapshot file name**</br>
-    Name of your SpatialOS Unreal GDK snapshot file.
+    Name of your snapshot file.
 
 * **Schema Generator**
 
     * **Generate Schema for all Supported Classes**</br>
```

Comment:
I'd suggest lower case for everything apart from the first word

Reply:
I've updated the code to reflect this.